src/components/filterdialog/filterdialog.js

```diff
@@ -145,6 +154,22 @@ function setVisibility(context, options) {
     if (options.mode === 'episodes') {
         showByClass(context, 'episodeFilter');
     }
+
+    if (!options.hasFilters) {
```

Comment:
Would it make sense to just call `getFilterStatus` here also instead of passing an `option`?